### DISCUSSION TITLE
error when escalating to Hive

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#5295] [Unsupported Versions] [answered]
After upgrading to v2.3.70 we now get errors when escalating alerts.  We get this red 400 warning.  
![Screenshot from 2021-08-24 08-19-12](https://user-images.githubusercontent.com/20424476/130634149-f859030d-ea3e-4882-bfb0-cdf64d55a2b8.png)

When trying to escalate an alert from Kibana it opens a new tab and then gives a '404 page not found' error.  As mentioned [here](https://github.com/Security-Onion-Solutions/securityonion/discussions/4323) the hive and soctopus logs don't seem to reveal clues.

I can log into the Hive web interface manually just fine.  I've checked the user's api key in Hive and it matches the 'hivekey' in global.sls. Is there another key I should be checking?

Also, I ran 'sudo soup' this morning to make sure any fixes were applied.  All went fine with no errors.  I even rebooted for good measure.  Still can't escalate to Hive.

**Comment by paintedrealms** [comment]
I too am getting the same issue for the same version number.  When I escalate i get the red bar with 400 error across the top of the screen.  Also, acknowledges give me a messages stating time to process and when I refresh, the issues are still in alerts.  the continue to be in alerts days later.

**Comment by paintedrealms** [comment]
is this the same as issue 5018?

**Comment by innovate-support** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/5437
